framework/doc/content/source/kernels/FunctionDiffusion.md

```diff
@@ -5,17 +5,17 @@
 ## Overview
 
 !style halign=left
-The `FunctionDiffusion` object represents a Laplacian operator term with a function coefficient for 
+The `FunctionDiffusion` object represents a diffusion term with a function coefficient for
 scalar field variables. This term is
 
 \begin{equation}
-  f(\mathbf{r}, t) \nabla^2 v
+  - \nabla \cdot f(\mathbf{r}, t) \nabla v
```

Comment:
I dont think so. it's missing a few terms for that to be true, see FunctionDiffusion.C

Reply:
nevermind!